From c8f4a9f75f12adb1f76391a838d75f2b427becb6 Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@fingolfin.org>
Date: Mon, 17 Jan 2022 01:38:56 +0000
Subject: [PATCH] 8555 vfstab(4) man incorrectly describes getvfsent(3C)
 Reviewed by: Yuri Pankov <ypankov@tintri.com> Reviewed by: Andy Fiddaman
 <andy@omnios.org> Approved by: Gordon Ross <gordon.w.ross@gmail.com>

---
 usr/src/man/man4/vfstab.4 | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/usr/src/man/man4/vfstab.4 b/usr/src/man/man4/vfstab.4
index f7b3d008b0..5b6a815784 100644
--- a/usr/src/man/man4/vfstab.4
+++ b/usr/src/man/man4/vfstab.4
@@ -64,7 +64,9 @@ This may be used when a field does not apply to the resource being mounted.
 .Pp
 The
 .Xr getvfsent 3C
-family of routines is used to read and write to
+family of routines is used to read
+.Pa /etc/vfstab .
+There are currently no library routines to automate the writing of
 .Pa /etc/vfstab .
 .Pp
 .Pa /etc/vfstab
-- 
2.40.1

